pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[project]
+name = "urotheliome-dash"
+version = "0.1.0"
+description = "Add your description here"
```

Comment:
Fill this in with something like "Dashboard for the Urotheliome research project"